src/testing/testing.ts

```diff
@@ -7,11 +7,12 @@ export function assertASTsAreEqual(
     actual: Node,
     context = "<root>",
     considerPosition = false
-) {
+): void {
     if (areSameType(expected, actual)) {
         if (considerPosition) {
             expect(
-                actual.position!.equals(expected.position!),
+                (actual.position == undefined && expected.position == undefined)
```

Comment:
Actually, wouldn't it be better to `expect(actual.position).to.eql(expected.position)`?